tools/crm_mon_print.c

```diff
@@ -1098,9 +1122,20 @@ print_html_status(pcmk__output_t *out, pe_working_set_t *data_set,
 
     /*** NODE LIST ***/
     if (is_set(show, mon_show_nodes)) {
-        out->begin_list(out, NULL, NULL, "Node List");
+        gboolean printed_header = FALSE;
+
         for (gIter = data_set->nodes; gIter != NULL; gIter = gIter->next) {
             pe_node_t *node = (pe_node_t *) gIter->data;
+
+            if (!str_in_unames(unames, node->details->uname)) {
+                continue;
+            }
+
+            if (printed_header == FALSE) {
+                printed_header = TRUE;
+                out->begin_list(out, NULL, NULL, "Node List");
+            }
+
```

Comment:
Don't we have to guard end_list() for html too?

Reply:
Yep, good catch.